test/arithmetic/fixed_to_float_test.dart

```diff
@@ -0,0 +1,166 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// fixed_to_float_test.dart
+// Test fixed point to floating point converters.
+//
+// 2024 October 24
+// Author: Soner Yaldiz <soner.yaldiz@intel.com>
+
+import 'dart:io';
+import 'dart:math';
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  test('Smoke', () async {
+    final fixed = FixedPoint(signed: true, m: 34, n: 33);
+    final dut = FixedToFloat(fixed, exponentWidth: 8, mantissaWidth: 3);
+    await dut.build();
+    File('${dut.name}.sv').writeAsStringSync(dut.generateSynth());
```

Comment:
do not leave these file-write operations in the test -- after test run there should be no temporary files remaining

Reply:
Should `dut.build()` be kept? Does it catch any issue that is missed by a test?

Reply:
It's fine to keep, it may help catch some issues like input port rule violations